app/models/manageiq/providers/google/cloud_manager/cloud_volume_snapshot.rb

```diff
@@ -1,2 +1,5 @@
 class ManageIQ::Providers::Google::CloudManager::CloudVolumeSnapshot < ::CloudVolumeSnapshot
+  supports :create
+  supports :update
+  supports :delete
```

Comment:
If we're going to say we support these operations we need to have the `raw_*` methods to implement them